### ISSUE TITLE
Unicornicopia sends a lot of email.

### ISSUE THREAD

**Issue by oatsandsugar** [cornelltech/unicornucopia-issue-tracker#5]
Maybe a likes digest, rather than an email for each like?

**Comment by AVatch** [comment]
Yup, we're working on a digest

**Comment by omris1** [comment]
I disagree that this is an enhancement, this is a bug. We're a captive audience that has to login to the app in any case, at best the notifications could be a cute addition, but right now it's notification hell and because the purpose of notifications is not clear I would suggest turning them off before changing them into a digest form.
